dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
 django-debug-toolbar==1.5     # https://django-debug-toolbar.readthedocs.io/en/stable/changes.html
+setuptools==33.1.1
```

Comment:
This issue describes the situation where an old version of pip clashes with the newer setuptools https://github.com/pypa/setuptools/issues/937

I think it would be preferable to upgrade our version of `pip` than to explicitly pin an old version of `setuptools`.